### PR TITLE
updated several files from Patch 1.0.0.290 (GS)

### PR DESCRIPTION
**Pull request by ihendriks**

### CODE REVIEW

**Comment by Azurency** [comment]
Great, thanks ! 👍 If you work on more file, push your changes on the same branch, I'll merge it all together

**Comment by ihendriks** [comment]
> Great, thanks ! 👍 If you work on more file, push your changes on the same branch, I'll merge it all together

I have some time this week, so might be able to push out a bunch of them.

**Comment by ihendriks** [comment]
Probably won't have time to do much more for now, but who knows.

File: Assets/UI/diplomacyactionview.xml [hunk 1]
```diff
@@ -154,7 +154,7 @@
   </Instance>
 
   <Instance Name="IntelOverviewDividerInstance">
-    <Box ID="Top" Anchor="C,C" Size="parent-20,1" Color="255,255,255,80"/>
+    <Box ID="Top" Anchor="TheDivide" Size="parent-20,1" Color="255,255,255,80"/>
```
Comment (Azurency) [inline Assets/UI/diplomacyactionview.xml:157]: There's an error here, it should be : 
```suggestion
    <Box ID="TheDivider" Anchor="C,C" Size="parent-20,1" Color="255,255,255,80" />
```

File: Assets/UI/ingame.lua [hunk 2]
```diff
@@ -1,12 +1,22 @@
--- ===========================================================================
+-- Copyright 2015-2018, Firaxis Games
 --	Civ6
+-- Root context for ingame (aka: All-the-things)
 --	Root context for ingame (aka: All-the-things)
+-- MODs / Expansions cannot use partial replacement as this context is
 -- ===========================================================================
+-- directly added to the UI Control Tree via engine.
```
Comment (Azurency) [inline Assets/UI/ingame.lua:7]: mmh weird beginning, should be : 
```
-- Copyright 2015-2018, Firaxis Games
-- Root context for ingame (aka: All-the-things)
-- MODs / Expansions cannot use partial replacement as this context is 
-- directly added to the UI Control Tree via engine.
```

File: Assets/UI/Instances/leadericon.lua [hunk 3]
```diff
@@ -50,55 +50,89 @@ end
...
-  
+function LeaderIcon:UpdateIcon(iconName: string, playerID: number, isUniqueLeader: boolean, ttDetails: string)
   local pPlayer:table = Players[playerID];
   local pPlayerConfig:table = PlayerConfigurations[playerID];
   local localPlayerID:number = Game.GetLocalPlayer();
-  local isHuman:boolean = pPlayerConfig:IsHuman();
 
   -- Display the civ colors/icon for duplicate civs
-  if(isUniqueLeader == false) then
+  if (isUniqueLeader == false and (playerID == localPlayerID or Players[localPlayerID]:GetDiplomacy():HasMet(playerID)))
```
Comment (Azurency) [inline Assets/UI/Instances/leadericon.lua:59]: Missing a `then` : 
```suggestion
  if isUniqueLeader == false and (playerID == localPlayerID or Players[localPlayerID]:GetDiplomacy():HasMet(playerID)) then
```

**Review by Azurency — COMMENTED** [review]
One or two things that are not well updated, otherwise it's all good. If you don't have the time to do the little update, I will do it myself after merging to `expansion2`.
I tested all that ingame, seems fine.

**Comment by Azurency** [comment]
Thanks for the help 😄👍, I'll merge it now so I can review the other pull request. It's only a few tweak remaining I'll do it.

**Comment by ihendriks** [comment]
Awesome. Was bound to have copied something over wrong, was going slightly crazy after a while to be quite honest haha.

**Pull request merged**